### ISSUE TITLE
Check documentation correctness on next (0.1.4) release.

### ISSUE THREAD

**Issue by eiri** [IBM/cloudant-python-sdk#276]
Deps PR #275 bumped sphinx to 5.0.0. There are some breaking changes, but as I squint on them they seems to not affect us. The CI was behaving, but nothing related to the change itself.

I'm merging it, but leaving a note to not forget to double-check generated documentation on release 0.1.4 to see if I missed anything and this update actually broke us.

**Comment by eiri** [comment]
I compared previous docs v0.1.3 with current doc build (sphinx 4.5.0 vs 5.0.2).

The changes are mostly cosmetic:

- Javascripts libs got bump. One interesting behavior difference is in search, now `ENABLE_SEARCH_SHORTCUTS` is `false` by default
- CSS styles got updated. If the new style an improvement is a subjective question, but at least docs are not unreadable. 
- One visual change is shortening response signature, e.g. previously they looked like `ibm_cloud_sdk_core.detailed_response.DetailedResponse` and now just `DetailedResponse`. Don't think this is a breaking change.
- Enums now seems to be listed in schema, e.g. I see a list with `<cite>lazy</cite>` included after `Attr str update` where that wasn't the case before, but that might be a specific to API changes. In any case that's not a breaking change.

tl;dr - I think we should be fine to release docs with sphinx 5.x, there are going to be changes from the previous formatting, but they are pretty minor to mandate pinning old sphinx version.
